build-system/test-configs/forbidden-terms.js

```diff
@@ -749,9 +749,6 @@ const forbiddenTermsGlobal = {
       'extensions/amp-render/1.0/storybook/Basic.js',
       'extensions/amp-selector/1.0/storybook/Basic.amp.js',
       'extensions/amp-selector/1.0/storybook/Basic.js',
-      'extensions/amp-sidebar/0.1/storybook/Basic.amp.js',
-      'extensions/amp-sidebar/1.0/storybook/Basic.amp.js',
-      'extensions/amp-sidebar/1.0/storybook/Basic.js',
```

Comment:
Plz restore these lines and the storybook examples, I know the name is confusing but it has nothing to do with Web Stories 😅

Reply:
Done, thanks!